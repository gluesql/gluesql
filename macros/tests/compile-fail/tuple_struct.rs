use gluesql_macros::FromGlueRow;

#[derive(FromGlueRow)]
struct T(i32);
