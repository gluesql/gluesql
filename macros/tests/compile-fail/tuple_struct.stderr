error: FromGlueRow supports only named fields
 --> tests/compile-fail/tuple_struct.rs:4:1
  |
4 | struct T(i32);
  | ^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compile-fail/tuple_struct.rs:4:15
  |
4 | struct T(i32);
  |               ^ consider adding a `main` function to `$DIR/tests/compile-fail/tuple_struct.rs`
